src/SAWScript/MGU.hs

```diff
@@ -246,25 +249,24 @@ instance UnifyVars Schema where
 -- NamedVars {{{
 
 class NamedVars t where
-  namedVars :: t -> S.Set Name
+  namedVars :: t -> M.Map Name Pos
```

Comment:
Similarly, it would be helpful to include a comment saying what the `Map` in the return type of `namedVars` represents.

Reply:
Agreed